Dockerfile

```
@@ -72,6 +72,9 @@ FROM runtime
 
 COPY --from=gobuild $BIN_PATH/$APP_NAME $BIN_PATH
 
+# Run as a user without privileges.
+USER nobody
```

Thoughts on naming the guy `keep`? Sounds more friendly to me 😸 

```suggestion
USER keep
```

---

😂 I think the `nobody` user is chosen for a particular reason though, see:
- https://en.wikipedia.org/wiki/Nobody_(username)
- https://unix.stackexchange.com/questions/186568/what-is-nobody-user-and-group